flake.nix

```diff
@@ -107,11 +107,20 @@
         # Don't build the manual as it's time consuming, and it requires fonts
         # that are not available in the sandbox due to internet connection
         # missing.
-        configureFlags = pkgs.lib.lists.remove "--with-manual" oldAttr.configureFlags;
+        configureFlags = [
+          "PDFINFO=false"
+        ] ++ (
+          pkgs.lib.lists.remove "--with-manual" oldAttr.configureFlags
+        );
         nativeBuildInputs = oldAttr.nativeBuildInputs ++ [
           pkgs.autoreconfHook
         ];
         buildInputs = [
+          # Build inputs added since release in nixpkgs
+          pkgs.cargo
+          pkgs.jq
+          pkgs.rustc
+        ] ++ [
```

Comment:
Potentially unpopular opinion: nix is a huge PITA. The “naïve” solution never works and i don't use it as all packages are continuously broken in some way. Docker is much easier and more inline with existing build processes for most packages. We may consider not wanting to take on the maintenance burden of Nix. But if maintainers think it important to support then please ignore me :)

Reply:
I agree it is a PITA but I personally get a lot of millage out of it in production. Even our own website build uses it because it is by far the easiest way to run multiple different versions in parallel. My approach is to support native builds first, containerized ones like Docker second, and finally specialty builds like Nix third. I don't plan to hold up the project for Nix support that might sometimes lag behind, but I do hope to keep supporting it.

@doronbehar I'm going to merge this PR to a develop branch so this can get tested along with other PRs we are targeting for v0.15.0. It would be nice if we could get the Flake support spun back up soon.